sw/airborne/subsystems/radio_control/cc2500_frsky/cc2500_paparazzi.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2019 Tom van Dijk <tomvand@users.noreply.github.com>
+ *
+ * This file is part of paparazzi.
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, write to
+ * the Free Software Foundation, 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ */
+
+#include "cc2500_paparazzi.h"
+
+#include "subsystems/radio_control.h"
+#include "peripherals/cc2500.h"
+#include "cc2500_common.h"
+#include "cc2500_frsky_common.h"
+#include "cc2500_settings.h"
+#include "cc2500_rx.h"
+
+#include <stdint.h>
+
+
+static uint16_t frsky_raw[RADIO_CTL_NB];
+
+
+#if PERIODIC_TELEMETRY
+#include "subsystems/datalink/telemetry.h"
+
+static void send_cc2500(struct transport_tx *trans, struct link_device *dev)
+{
+  pprz_msg_send_CC2500(trans, dev, AC_ID,
```

Comment:
what about reusing a message already existing ? https://github.com/paparazzi/pprzlink/pull/118#issuecomment-651780344

Reply:
Agreed, will swap it for the PPM message (provided that this does not cause conflicts with other modules/subsystems).